static-site/src/pages/groups.jsx

```diff
@@ -115,6 +115,7 @@ class GroupsPage extends React.Component {
 
         <ListResources
           resourceType="dataset"
+          defaultGroupLinks={true}
```

Comment:
[crystal ball gazing -- not a request for changes]

Implicit in the original design is that card titles will become links by simply adding the card name to the current URL. It works well for the groups page (e.g. `groups` + `blab`), but I don't think it'll work for an individual groups page.¹ We'll probably just disable the links for those pages.

Also, for what it's worth, the `={true}` can be dropped, if you prefer. 

¹ Depends on how card titles are chosen of course, but the blab group will probably have a "229e" card and [/groups/blab/229e](https://next.nextstrain.org/groups/blab/229e) is 404.

Reply:
> Depends on how card titles are chosen of course, but the blab group will probably have a "229e" card and [/groups/blab/229e](https://next.nextstrain.org/groups/blab/229e) is 404.

I would think on the individual groups page, the titles would not include `groups/<group_name>`. The `resourceListingCallback` would just return `pathPrefix: "groups/<group_name>"`? (Maybe I'm completely off here, still catching up on all the ListResources work)